src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -121,9 +121,7 @@ public static void Main(string[] args)
 
             // Custom Service Initialization
             //log.Info("Adding Services");
-            engine.Services.Add(new SettingsService(false));
             engine.Services.Add(new ExtensionService());
-            engine.Services.Add(new ProjectService());
```

Comment:
Can the SettingsService and ProjectService classes also be deleted?

Reply:
No. They are used when the engine is used by the runner. It's only the agent that uses a "downgraded" version of the engine.